### DISCUSSION TITLE
신입 백엔드 개발자 최소 기준

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#14] [Q&A] [answered]
JWT
CRUD REST API
Swagger
ERD
AWS - EC2, RDS
RESR API 설계
Docker
Front-end (React)

**Comment by tjdud22** [comment, marked as answer]
JWT(Json Web Token)는 말그대로 웹에서 사용되는 JSON 형식의 토큰에 대한 표준 규격. 주로 사용자의 인증(authentication) 또는 인가(authorization) 정보를 서버와 클라이언트 간에 안전하게 주고 받기 위해서 사용됩니다.
